longest-run.playground/Contents.swift

```diff
@@ -7,7 +7,30 @@ This excercise lends itself to TDD and we have provided a few basic tests below.
 import Cocoa
 
 func longestRun(whole: String) -> String {
-    return whole
+    let sorted = whole.characters.sort()
```

Comment:
Duh! this does not give the longest contiguous run, but the most frequent character... oh well...